spec/integration/data_stream/_dev/benchmark/pipeline/config.spec.yml

```diff
@@ -0,0 +1,7 @@
+spec:
+  type: object
+  additionalProperties: false
+  properties:
+    num_docs:
```

Comment:
Should this field be mandatory?

Reply:
It has a default value if not present, so should not break things to have the file empty, apart from being useless. I can make it mandatory if that makes more sense to you. Or maybe we can set a spec constraint of "should have at least one", this way if/when more options are added would be enough to make sure it is not an empty file.

Reply:
I added the `minProperties` and all tests failed with a message such as 

```
                Error Trace:    validator_test.go:302
                Error:          []string{"could not read root folder spec file: could not load spec for \"integration/data_stream/spec.yml\": could not load spec for \"integration/data_stream/_dev/spec.yml\": could not load spec for \"integration/data_stream/_dev/benchmark/spec.yml\": could not load spec for \"integration/data_stream/_dev/benchmark/pipeline/spec.yml\": could not load schema for \"integration/data_stream/_dev/benchmark/pipeline\": failed to load schema for \"integration/data_stream/_dev/benchmark/pipeline/config.spec.yml\": minProperties must be of an integer"} does not contain "field \"event.dataset\" is defined multiple times for data stream \"wrong\", found in: ../../../../test/packages/bad_duplicated_fields/data_stream/wrong/fields/base-fields.yml, ../../../../test/packages/bad_duplicated_fields/data_stream/wrong/fields/ecs.yml"
```

even the ones without that file, I guess is an issue with the validation or schema generation, I will make it mandatory for now.